### DISCUSSION TITLE
Explorer seems to not be showing archival results, does this affect RPC calls?

### DISCUSSION THREAD

**Discussion by mikedotexe** [near/near-explorer#496] [Q&A] [answered]
I visited this section of Explorer recently:
https://explorer.testnet.near.org/accounts/mike.testnet

This is an account that has interacted with many contracts, but the old transactions don't seem to be listed. I believe I've overheard that the Explorer is no longer archival, so it's expected behavior to not see old transactions.

Then I looked in my bash history and saw I had this NEAR CLI command:

`near tx-status 8gZtQFrCLZRgpY4PVL6BN7iUUFkKuks2DBZeC3geRmsr --accountId mike.testnet`

This returns:

>Transaction 8gZtQFrCLZRgpY4PVL6BN7iUUFkKuks2DBZeC3geRmsr doesn't exist…

So it seems that the RPC call is also non-archival? First I want to check my assumptions here, second I want to be able to answer the question that may come from a community member or partner: "How can I see transaction details from several epochs ago?"

**Comment by frol** [comment, marked as answer]
@mikedotexe Great question! There are many things in play.

First, this particular account in question (`mike.testnet`) has indeed been involved only in 2 transactions in the **current** testnet network. testnet has had a hard-reset a few weeks ago (we lose all the history when we do hard-reset, but the accounts, contracts, and their state are preserved, and we also keep the block height, though you cannot really request any block before the block height the genesis started).

Second, Explorer uses its dedicated archival node to serve all the data (including historical), and it is working fine now. Wallet, on the other hand, relies on the public rpc.*.near.org RPC nodes, which are not archival since we cannot scale archival nodes (each archival node requires hundreds of GB of storage). This is why you cannot get the status of the transaction via RPC (while Explorer displays the status just fine).

Getting back to the original question: "How can I see transaction details from several epochs ago?"

The current answer is: you are required to run your own archival node to be able to query the arbitrary historical data. We are considering to provide an archival node for RPC usage with no guarantees about its performance/availability (we don't want to become a single point of failure in the decentralized world).
